gprofiler/metadata/application_identifiers.py

```diff
@@ -290,6 +290,21 @@ def get_app_id(self, process: Process) -> Optional[str]:
         return None
 
 
+class _RubyModuleApplicationIdentifier(_ApplicationIdentifier):
+    def get_app_id(self, process: Process) -> Optional[str]:
+        skip_next = False
+        for arg in process.cmdline():
+            if skip_next:
+                skip_next = False
+                continue
+            if arg == "-r":
```

Comment:
I read that these 2 are equal:
```
ruby -r require.rb app.rb
ruby -rrequire.rb app.rb
```

so you need to check if anything follows the `-r` and if so, don't skip next and only skip the current argument.